backend/mora/service/configuration_options.py

```diff
@@ -56,7 +56,7 @@ def set_org_unit_configuration(unitid):
                 query = "UPDATE orgunit_settings set value=? where id=?"
                 cur.execute(query, (value, rows[0][0]))
             else:
-                raise('Non-consistent settings for {}'.format(unitid))
+                raise Exception('Non-consistent settings for {}'.format(unitid))
```

Comment:
FWIW we have a variety of (somewhat) documented error codes in `mora.exceptions`. You'd use them like this:

```python
exceptions.ErrorCodes('Inconsistent settings for unit', org_unit_uuid=unitid)
```